Overview.bs

```diff
@@ -437,7 +428,7 @@ The inputs to this algorithm are:
 
 *  <var>font subset</var>: an [=incremental font|incremental=] [=font subset=].
 
-*  <var>initial font subset URI</var>: an [[rfc3986#section-4.3|absolute URI]] which identifies the location of the initial incremental
+*  <var>initial font URL</var>: a [[whatwg-url#concept-url|URL]] which identifies the location of the initial incremental
```

Comment:
Good. "initial font subset" implied that there was, somewhere, also an initial font which was not subsetted; this is clearer